pkg/apis/machine/v1beta1/machinehealthcheck_types.go

```diff
@@ -99,7 +99,6 @@ type MachineHealthCheckStatus struct {
 	ExpectedMachines *int `json:"expectedMachines"`
 
 	// total number of machines counted by this machine health check
-	// +kubebuilder:default=0
```

Comment:
Is this not allowed anymore?

Reply:
nope, API machinery complains. Note this never applied since defaulting requires structural schema to work.